index.html

```diff
@@ -0,0 +1,290 @@
+        <a class="block" href="#">
+          <li>ruby</li>
+        </a>
+      </ul>
+      <a class="see-project" href="#"> See Project </a>
+    </section>
+
+    <section class="project-cards">
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
```

Comment:
I suggest for these, make sure they match the figma design. This is how your project currently looks:
<img width="496" alt="Screenshot 2022-12-02 at 14 43 13" src="https://user-images.githubusercontent.com/48546992/205285747-1f50875d-ae33-4dd7-ba72-ebf89d8eb108.png">
this is how the figma design looks
<img width="375" alt="Screenshot 2022-12-02 at 14 32 44" src="https://user-images.githubusercontent.com/48546992/205285912-d2928eb0-8722-49d8-b93d-41da672a182e.png">